Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -445,42 +442,43 @@ def getJobOutput(self, jobID, _localDir=None):
     # same machine
     # workingDirectory = self.ceParameters.get( 'WorkingDirectory', DEFAULT_WORKINGDIRECTORY )
 
-    if not self.useLocalSchedd:
-      iwd = None
+    iwd = None
 
-      # TOREMOVE: once v7r0 will mainly be used, remove the following block that was only useful
-      # when path to output was not deterministic
-      status, stdout_q = commands.getstatusoutput('condor_q %s %s -af SUBMIT_Iwd' % (self.remoteScheddOptions,
+    # TOREMOVE: once v7r0 will mainly be used, remove the following block that was only useful
```

Comment:
Can this be removed now?

Reply:
Well I don't know.
If someone jumps from `v6` to `v7r1` directly, there will be some issues if we remove this block.
From what I understand, known VO using DIRAC are now using `v7`. If this is the case, then I guess we can remove it.
Else, we can delete it in another PR targeting `v7r2` or `v7r3`.
What do you think?
Any opinion @fstagni?

Reply:
I think it can be removed now.